### ISSUE TITLE
Add a story arc type and associated subtypes

### ISSUE THREAD

**Issue by rhdunn** [schemaorg/schemaorg#3665]
The use case here is to model a group of episodes or works that form a single story arc within a series or season. Examples include:
- Story arcs in comics and manga over several issues (stories) that when bundled into volumes may span volumes, e.g. [One Piece Story Arcs](https://onepiece.fandom.com/wiki/Story_Arcs);
- Podcasts of RPG playthroughs (game scenarios) where multiple scenarios are played in a season and over multiple episodes;
- TV shows such as adaptations of Manga;
- Radio dramas.

NOTE: These can be non-consecutive e.g. in the case of manga arcs, e.g. adding interludes between long arcs or at the end of a volume.

This would have the same properties as `CreativeWorkSeries` and the following hierarchy:
```
- CreativeWork
  - StoryArc
    - ComicStoryArc
    - PodcastStoryArc
    - RadioStoryArc
    - TVStoryArc
```

**Comment by MatthiasWiesmann** [comment]
Could you please explain why we need a different type from `CreativeWorkSeries`? 
What are the semantic differences?

**Comment by rhdunn** [comment]
> A *story arc* is an individual story (with a beginning, middle, and end) that has a start and end date that spans one or more episodes/issues. It may be a subset of a given season/volume, may span multiple seasons/volumes, or may just span several issues if those issues are not bundled into a volume.

For episodic works -- podcast, radio show, TV show -- there is currently:
1. the *series* refers to the specific show -- e.g. "One Piece" the anime;
2. a *season* refers to a consecutive run of episodes;
3. an *episode* refers to a specific episode in the series.

For comics there is currently:
1. the *series* referring to the entire comic production -- e.g. "Batman", "One Piece", etc.
2. a *volume* being a collection of issues, like with manga;
3. an *issue* being an individual comic release -- this may be separate, or within a volume.

* The One Piece link I referenced above showed the structure relating to story arcs.
* For batman, spiderman, etc. a story arc could be their origin story across multiple issues.
* For podcasts, I'm thinking of things like https://the-apocalypse-players.fandom.com/wiki/The_Scenarios where each season has multiple scenarios (playthroughs of specific RPG campaigns).
* For TV you could have things like Stargate season 1 episode 20-21 and season 2 episode 1 as a single story arc.

**Comment by MatthiasWiesmann** [comment]
To summarize, you are suggesting a new relation between say [TVEpisode](https://schema.org/TVEpisode) instances and some object that groups, in the same way they are part of a season, they would be part of a StoryArc? 

If this is indeed the case, I would suggest that the StoryArc is separate from any media type (I'm sure we can find cases where a story arc spans multiple medias). 

As I see it, a StoryArc would be its own CreativeWork with just a collection of parts which can be any CreativeWork. 

Does this make sense?

**Comment by rhdunn** [comment]
Yes I am and yes that makes sense. I proposed several subclasses of `StoryArc` for media-specific story arcs in the issue description like there are for seasons, episodes, etc.

**Comment by MatthiasWiesmann** [comment]
But what is the point of media-specific story arcs, as they can span multiple medias and I don't see media specific attributes?

**Comment by MatthiasWiesmann** [comment]
OK, do you have a concrete application in mind which will consume that data?
